### PR TITLE
SettingsNode: implement multiple remote nodes support

### PR DESCRIPTION
**Pull request by xiphon**
Closes https://github.com/monero-project/monero-gui/issues/1008

monero-gui-multiple-remote-nodes

### CODE REVIEW

**Comment by benevanoff** [comment]
From my tests it seems that the app doesnt remember which node was used last after closing/reopening, currently just falling back to whichever is at the front of the list. It'd be nice if either the last node used was stored persistently or if it were possible to set a default node to be used at startup.

**Comment by xiphon** [comment]
> the app doesnt remember

Updated.

**Comment by benevanoff** [comment]
Awesome!

**Review by benevanoff — APPROVED** [review]

File: pages/settings/SettingsNode.qml [hunk 1]
```diff
@@ -262,80 +262,94 @@ Rectangle{
...
-                checked: persistentSettings.is_trusted_daemon
-                onClicked: {
-                    persistentSettings.is_trusted_daemon = !persistentSettings.is_trusted_daemon
-                    currentWallet.setTrustedDaemon(persistentSettings.is_trusted_daemon)
-                }
-                text: qsTr("Mark as Trusted Daemon") + translationManager.emptyString
-            }
+                        Rectangle {
+                            anchors.fill: parent
+                            anchors.topMargin: 5
```
Comment (selsta) [inline pages/settings/SettingsNode.qml:305]:
```suggestion
```

Destroys the vertical text alignment, unless there is a reason for it I didn’t figure out.

**Comment by selsta** [comment]
Looks a bit broken on first startup

Screenshot 2021-04-05 at 20 59 17

**Comment by xiphon** [comment]
Updated.

**Review by selsta — APPROVED** [review]

**Pull request merged**